apps/meeting/serverless/src/handlers.js

```diff
@@ -1,11 +1,25 @@
 // Copyright 2020-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: MIT-0
 
-var AWS = require('aws-sdk');
+const AWS = require('aws-sdk');
 var ddb = new AWS.DynamoDB();
 const chime = new AWS.Chime({ region: 'us-east-1' });
 chime.endpoint = new AWS.Endpoint('https://service.chime.aws.amazon.com/console');
 
+// Create ans AWS SDK Chime object. Region 'us-east-1' is globally available.
+const chimeRegional = new AWS.ChimeSDKMeetings({ region: 'us-east-1' });
+const chimeRegionalEndpoint = process.env.REGIONAL_ENDPOINT || 'https://meetings-chime.us-east-1.amazonaws.com';
```

Comment:
Could we make it like below to reduce line width?
```js
const chimeRegionalEndpoint =
  process.env.REGIONAL_ENDPOINT ||
  "https://meetings-chime.us-east-1.amazonaws.com";
```

Reply:
done